.github/workflows/ci.yml

```diff
@@ -5,7 +5,8 @@ on:
     branches:
       - main
   pull_request:
-    paths: ['!docs/**']
+    paths-ignore:
+      - docs/**
```

Comment:
Not related to this PR but I noticed that the last commits I've pushed didn't trigger any CI execution.

Reply:
Fixed in https://github.com/grafana/explore-profiles/pull/77